README.md

```diff
@@ -1,16 +1,10 @@
-# AND101 - Introduction to Xamarin.Android 
+# AND101 - Introduction to Xamarin.Android
 
-This repository contains exercises for the Xamarin University [AND101 - Introduction to Xamarin.Android ](http://university.xamarin.com/classes/#and101-intro-to-xamarin-android) training course.
+This repository contains exercises for the Xamarin University **AND101 - Introduction to Xamarin.Android** training course materials. This course is now retired and this content is no longer being actively maintained.
 
-Xamarin University offers live and self-guided training on app development for Windows, iOS, and Android. Our self-guided courses are free and let you earn credit towards your Xamarin certification. You can upgrade to a full Xamarin University subscription to get unlimited live instructor-led courses from our extensive catalog and take a certification exam. Please visit [Xamarin University](https://university.xamarin.com/) to learn more or [start a trial subscrption](https://www.xamarin.com/university#request-trial).
+All the Xamarin University courses are currently retired. For up-to-date Xamarin training, you can start earning badges taking the latest [Microsoft Learn modules about Xamarin](https://aka.ms/learn-xamarin).
```

Comment:
Remove the "All the Xamarin University..." sentence and just make it with "For the latest free training on Xamarin, head over to [Microsoft Learn](https://aka.ms/learn-xamarin)."